can_tools/scrapers/official/ME/me_vaccines.py

```diff
@@ -460,13 +456,19 @@ def normalize(self, resjson: dict) -> pd.DataFrame:
 
         # combine into dataframe
         df = pd.DataFrame.from_records(data_rows)
-        # they report j&j doses in either the initiated or completed column--the other is empty (NA)
-        df["jj_complete"] = df["jj_complete"].fillna(0)
-        df["jj_init"] = df["jj_init"].fillna(0)
+
+        # J&J doses are reported in either the initiated or completed column--the other is empty (NaN)
+        # NOTE(sean): 11/14/21, the 5-11 age group does not have J&J data (it's not authorized for those under 18),
+        # but the J&J columns need to exist for the rest of the method to work.
+        # This just adds an empty column for if the J&J columns do not exist.
+        if "jj_complete" not in df.columns:
+            df["jj_complete"] = 0
+        if "jj_init" not in df.columns:
+            df["jj_complete"] = 0
 
         # format, calculate total_vacccine_initiated + map CMU
         out = (
-            df.dropna()  # counties that don't have age 0-11 return NA for value
+            df.fillna(0)
```

Comment:
I don't remember why we were dropping values NA values here, but I think it makes more sense to just mark them as 0s (especially since we were already doing that with the J&J doses above)